### PR TITLE
Simplify code with use conditional invoke

### PR DESCRIPTION
**Pull request by SuperNova911**
Contributes to [dotnet/docs#15768](https://github.com/dotnet/docs/issues/15758)

Use conditional invoke to simplify code

### CODE REVIEW

File: snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs [hunk 1]
```diff
@@ -228,8 +228,7 @@ public class Publisher
         protected virtual void RaiseSampleEvent()
         {
             // Raise the event by using the () operator.
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs]: I like this change. We should change the comment to match the new code: 

```suggestion
            // Raise the event in a thread-safe manner using the ?. operator.
```

Comment (SuperNova911) [reply snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs]: You're right. How did I miss that XD

**Comment by BillWagner** [comment]
Thanks @SuperNova911 I'll :shipit: now.

File: snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs [hunk 2]
```diff
@@ -227,7 +227,7 @@ public class Publisher
         // to enable derived classes to raise the event.
         protected virtual void RaiseSampleEvent()
         {
-            // Raise the event by using the () operator.
+            // Raise the event in a thread-safe manner using the ?. operator.
```
Comment (idbrii) [inline snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs]: How is this threadsafe? Isn't it just null-safe?

**Pull request merged**